docs/framework/network-programming/httplistener.md

```diff
@@ -9,15 +9,15 @@ ms.assetid: 5b89d3fb-3c9a-49e2-af1f-c34c020c68ac
 The <xref:System.Net.HttpListener> class provides a programmatically controlled HTTP protocol listener. The listener is active for the lifetime of the <xref:System.Net.HttpListener> object and runs within your application.  
   
 ## HTTP.SYS  
```

Comment:
These changes will clash with my removal of this article in #15481

Reply:
what do i do in this case?

Reply:
@eddynaka Just revert this file to its original state.

Reply:
@Thraka should i revert just the download part or the entire file? Thanks